@navikt/core/react/src/util/hooks/descendants/descendant.stories.tsx

```diff
@@ -126,9 +126,11 @@ function Input() {
       inputMode="numeric"
       onKeyDown={(event) => {
         if (event.key === "ArrowRight") {
+          event.preventDefault();
           descendants.next(index, false)?.node.focus();
```

Comment:
With next (and prev) multiple instances i would argue that "loop" should be false to avoid it jumping from bottom -> start or start -> bottom with arrowDown/Up
```
descendants.next(..., false)
```

Reply:
It seemed google search does the same behaviour (loops around), but maybe it's not that common actually 🤔

Reply:
Could just be a prop, example combobox wants it off, but maybe a search wants it on 🤔